backend/apid/middlewares/authorization_attributes.go

```diff
@@ -187,6 +89,12 @@ func (a LegacyAuthorizationAttributes) Then(next http.Handler) http.Handler {
 			attrs.Verb = "list"
 		}
 
+		// Add the user to the attributes
+		if err := getUser(ctx, attrs); err != nil {
+			http.Error(w, err.Error(), http.StatusUnauthorized)
```

Comment:
If getUser is hitting the DB then the error should be inspected a bit further before being displayed. If there are I/O errors the server should probably return 503.